platform-sdk/docs/proposals/TSS-Library/TSS-Library.md

```diff
@@ -268,34 +276,47 @@ Given the input:
 9. If `lhs` != `rhs`: Return False
 10. `inner` = 0
 11. For each pair (`i`:index ; `k`:commitment value)  in `statement.polynomial_commitment`:
-    
+
     11.1.  `term` = For each `sid` in `statements.sids`: sum (`sid` raised to the power of `i`) multiplied by (`x` raised to the power of `sid`)
 
     11.2.  `inner` += `k` multiplied by `term`
+
 12. `lhs` = (`inner` multiplied by `x'`) add `proof.A`
+
```

Comment:
Did we intend to break the ordered list here?  These extra lines make the spacing of the list inconsistent and a bit stutter-ish.